### PR TITLE
Adding Connehito/cakephp-master-replica

### PR DESCRIPTION
**Pull request by o0h**
https://packagist.org/packages/connehito/cakephp-master-replica

[Question]
I've Listed in _ORM and Datamapping_  according to the exists categories.
But this plugin doesn't provide a data-mapping feature, works in DBO layer.
Is it better if I add a new category like _Database_ and put this on it?

### CODE REVIEW

**Comment by ADmad** [comment]
We can just rename `ORM and Datamapping` to `ORM / Database / Datamapping`.

**Comment by o0h** [comment]
Thanks!
Rename and add a commit.

**Comment by dereuromark** [comment]
The header must also be aligned for quick jumping.

File: README.md [hunk 1]
```diff
@@ -308,6 +308,7 @@ Additional lists you might find useful:
 - :strawberry: [RowLocker plugin](https://github.com/lorenzo/row-locker) - Exclusive locks for rows in your tables.
 - [Serializeable Data Types plugin](https://github.com/burzum/cakephp-serialize-data-types) - Serialize DB content as JSON or using phps serializing functions.
 - :strawberry: [Muffin/Webservices ORM plugin](https://github.com/usemuffin/webservice) - An ORM like interface for webservices.
+- :strawberry: [Connehito/Master Replica Plugin](https://github.com/Connehito/cakephp-master-replica) - Switch master/replica database connections.
```
Comment (dereuromark) [inline README.md]: The plugin name seems to be `Connehito\\CakephpMasterReplica` => `Connehito\\CakephpMasterReplica plugin` as per guidelines.

**Comment by o0h** [comment]
Thank you so much for your advice!
As you pointed out, I have changed the name of the plugin in the README. https://github.com/FriendsOfCake/awesome-cakephp/pull/359/commits/6493242dee9ca6316b808235d9c84e807cc3434c

Is there anything else that needs to be changed? I'd appreciate it if you could tell me where to fix it.

**Pull request merged**